sdk/digitaltwins/azure-digitaltwins-core/src/test/java/com/azure/digitaltwins/core/QueryTests.java

```diff
@@ -29,7 +29,7 @@
     @Override
     public void validQuerySucceeds(HttpClient httpClient, DigitalTwinsServiceVersion serviceVersion) {
         DigitalTwinsClient client = getClient(httpClient, serviceVersion);
-        int pageSize = 5;
+        int pageSize = 3;
```

Comment:
why is this changed?

Reply:
this change doesn't really matter, I changed it during fixing this test.

Reply:
it just changes the max-item-per-page to 3 from 5 ...